gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/main/java/io/gravitee/am/gateway/handler/uma/resources/endpoint/ResourceAccessPoliciesEndpoint.java

```diff
@@ -73,9 +73,9 @@ public void list(RoutingContext context) {
         final String resource = context.request().getParam(RESOURCE_ID);
 
         subjectManager.findUserIdBySub(accessToken)
-                .flatMapSingle(userid -> resourceService.findAccessPolicies(domain.getId(), client.getId(), userid, resource)
-                .map(AccessPolicy::getId)
-                .toList())
+                .flatMapSingle(userid -> resourceService.findAccessPolicies(domain.getId(), client.getId(), userid.id() /*todo degraded mode: should we use full id here as well?*/, resource)
```

Comment:
id() is fine. This entity is currently managed only in the CP (not in the DP) so it is not possible to create it (or access it) without a valid user internalId